modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -241,8 +241,8 @@
                view-nm))
           (let [table-id (t2/select-one-pk Table :db_id (u/the-id database), :name view-nm)]
               ;; and its columns' :base_type should have been identified correctly
-            (is (= [{:name "numeric_col",   :database_type "numeric(10,2)",         :base_type :type/Decimal}
-                    {:name "weird_varchar", :database_type "character varying(50)", :base_type :type/Text}]
+            (is (= [{:name "numeric_col",   :database_type "numeric",         :base_type :type/Decimal}
```

Comment:
any idea why we lose the precision here?

Reply:
The subset of the implementation of `getColumns` we used previouisly uses `pg_type.typbasetype`. See
https://github.com/aws/amazon-redshift-jdbc-driver/blob/master/src/main/java/com/amazon/redshift/jdbc/RedshiftDatabaseMetaData.java#L2365C24-L2365C35
But now we're using `svv_all_columns.data_type`, which stores the precision in a separate column. 
See https://docs.aws.amazon.com/redshift/latest/dg/r_SVV_ALL_COLUMNS.html

What gives me confidence that these are roughly equivalent is that the JDBC driver implementation uses either, depending on the pattern of schemas that you select from. For example, if you select a schema pattern that corresponds to a "LOCAL_SCHEMA_QUERY", it will use `pg_type.typbasetype`, but if you specify a schema pattern that corresponds to a `NO_SCHEMA_UNIVERSAL_QUERY` it can use `svv_all_columns.data_type`.